### PR TITLE
HW19

### PR DESCRIPTION
**Pull request by AmalShalabi**
Hi Elias !
This is My HW19.
What can I say!? no doubt, we'll be able to solve the exercises in less than 30 minutes JUST after your good teaching ! 
Because you always care to make this more interesting  ;) 

Waiting for your review!

Amal

### CODE REVIEW

File: HW19/js/script.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+ var counter = 0, sum = 0;
+
+function gcollege() {
+    let data = [];
+    data[0] = document.getElementById("sname");
+    data[1] = document.getElementById("sgrade");
+    Avg_s= document.getElementById("avg_students");
+    Count_s = document.getElementById("num_students");
```
Comment (EliasSalom) [inline HW19/js/script.js]: I don't see any declaration of any variable of `Avg_s` and `Count_s `

File: HW19/js/script.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+ var counter = 0, sum = 0;
+
```
Comment (EliasSalom) [inline HW19/js/script.js:2]: I don't see any update to use the `local storage` !!!

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please check my comment

File: HW19/js/script.js (hunk 1 above)
Comment (AmalShalabi) [reply HW19/js/script.js]: Hey Elias, 
I did the declaration to the missing variables.
The code works ,  that's why I didn't notice..

File: HW19/js/script.js (hunk 2 above)
Comment (AmalShalabi) [reply HW19/js/script.js:2]: I created a new repo (hw20)  where the updated code 
And attached new CSS file which differs from  the CSS file that is already included in hw19.

**Comment by AmalShalabi** [comment]
Hey Elias , I geuss I fixed all the errors in the code
if everything is ok , please confirm merge

**Review by EliasSalom — APPROVED** [review]
Good work

**Pull request merged**